star-ros2/src/star_bringup/README.md

```
@@ -0,0 +1,145 @@
+Starts `star_safety_monitor` in lifecycle-managed mode via `nav2_lifecycle_manager`.
+Monitors heartbeat and motor stall; publishes `/emergency_stop` and `/diagnostics`.
+
+```bash
+ros2 launch star_bringup safety_monitor.launch.py
+```
+
+---
+
+## Configuration Files
+
+### `config/ekf.yaml`
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)